superdesk/storage/amazon/amazon_media_storage.py

```diff
@@ -61,7 +61,8 @@ def url_for_media_default(app, media_id, content_type=None):
     extension = str(_guess_extension(content_type)) if content_type else ''
     url = '%s.%s/%s%s' % (app.config['AMAZON_CONTAINER_NAME'], endpoint, media_id, extension)
```

Comment:
extension won't work without proxy so you have to add it under `if app.config.get('AMAZON_PROXY_SERVER'):`